cfgov/jinja2/v1/_includes/organisms/info-unit-group.html

```diff
@@ -18,7 +18,7 @@
 
    ========================================================================== #}
 
-{% macro info_unit_group(blocks=[], columns=2, heading) %}
+{% macro info_unit_group(heading, blocks=[], columns=2) %}
```

Comment:
Order of this code comment should be changed to reflect this code change https://github.com/cfpb/cfgov-refresh/blob/a8343f647824b6ad6aadb35db35fa5d991b6c3a2/cfgov/jinja2/v1/_includes/organisms/info-unit-group.html#L16

Reply:
Good call! Done.